lib/components/DataTable.js

```diff
@@ -2,12 +2,16 @@ import React, {PropTypes} from 'react';
 
 export default class Admin extends React.Component {
   static propTypes = {
-    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    data: PropTypes.arrayOf(PropTypes.object),
     showHeaders: PropTypes.bool
   }
 
   render() {
     const {data} = this.props;
+    if (!data || data.length < 1) {
```

Comment:
doesn't this belong in a `componentWillReceiveProps(){}`?
Then again, maybe it's a thing we can do at the router later.